packages/tempo/src/plugins/tempo.plugin.ts

```
@@ -75,24 +73,168 @@ const SCHEMA = [
+	if (term) {
+		const range = term.define.call(tempo, false);
+		if (range && range.start && range.end) {
+			const start = range.start.toDateTime();
+			const end = range.end.toDateTime();
+
+			switch (anchor) {
+				case 'start':
+					return start;
+
+				case 'mid':
+					const midNano = (start.epochNanoseconds + end.epochNanoseconds) / 2n;
```

Biome (2.4.9)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)